File: BE/src/main/java/com/self/roomescape/entity/Community.java
package com.self.roomescape.entity;


import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;

import javax.persistence.*;
import java.time.LocalDateTime;

@Setter
@Getter
@Entity
@EntityListeners(AuditingEntityListener.class)
@Table(name = "community")
public class Community {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "communityId")
    private Long rid;

    @ManyToOne
    @JoinColumn(name = "uid")
    private User user;

    private String title;

    private String content;

    private String header;

    @CreatedDate
    @Column(nullable = false)
    private LocalDateTime createDate;

    @LastModifiedDate
    @Column(nullable = false)
    private LocalDateTime updateDate;
}

File: AOS/app/src/main/java/com/ssafy/moabang/data/model/dto/Community.kt
package com.ssafy.moabang.data.model.dto

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.io.Serializable
import java.time.LocalDateTime

@Parcelize
data class Community(
    var count : Int, // 댓글 수
    var rid : Int,
    var user : User,
    var title : String?,
    var content : String?,
    var createDate : LocalDateTime,
    var updateDate : LocalDateTime,
    var header : String?,
    var reportCnt : Int,
) : Parcelable
// LocalDateTime